cluster-autoscaler/core/utils.go

```diff
@@ -48,6 +49,20 @@ const (
 	ReschedulerTaintKey = "CriticalAddonsOnly"
 )
 
+var (
+	nodeConditionTaints = taintKeySet{
```

Comment:
It would be great to automatically extend the set if new taits are defined by scheduler. 
Can we list them somehow - instead of enumerating each separately?

Reply:
Agreed, I didn't like writing this. Unfortunately the scheduler api (where these are defined) only defines these a string constants. Unless they/we add code in that repo, the only solution I could see would be to ignore the entire key prefix.

Reply:
those prefixes for your reference would be:
node.kubernetes.io/.*
node.cloudprovider.kubernetes.io/.*

Reply:
Seems to vast. Let's leave it.